docs/UserGuide.md

```diff
@@ -281,29 +310,36 @@ If you have completed prescriptions, you will see something similar to this.
 
 ### How to edit a prescription : `edit`
 
-If you made a mistake and would like to make some changes to your entry, type the following command.
+You may wish to update any of the fields of the prescription for a variety of reasons, such as a change in dosage, or an increase in stock. BayMeds allows you to edit such fields in the prescription.
+
+In order to do so, type the following command.
 
 Format:
 ```
 edit
-  INDEX
-  [mn/NAME]
-  [d/DOSAGE]
-  [f/FREQUENCY]
-  [sd/START DATE]
-  [ed/END DATE]
-  [exd/EXPIRY DATE]
-  [ts/TOTAL STOCK]
-  [n/NOTE]
+  <index>
+  [mn/<medication_name>]
+  [d/<dosage>]
+  [f/<frequency>]
+  [sd/<start_date>]
+  [ed/<end_date>]
+  [exd/<expiry_date>]
+  [ts/<total_stock>]
+  [n/<note>]
 ```
 
-This edits the prescription at the specified `INDEX`. The index refers to the index number shown according to [list](#listing-all-prescriptions--list).
+Based on fields you specify, the corresponding fields of the index-identified prescription will be updated with the new value.
 
```

Comment:
We aren't able to edit completed prescriptions right? Would this be misleading then.